src/openforms/emails/tests/test_tasks_integration.py

```diff
@@ -48,18 +49,26 @@ def check_config(self):
         raise InvalidPluginConfiguration("Invalid")
 
 
-@patch(
-    "openforms.emails.tasks.GlobalConfiguration.get_solo",
-    return_value=GlobalConfiguration(
-        recipients_email_digest=["tralala@test.nl", "trblblb@test.nl"]
-    ),
-)
 @override_settings(LANGUAGE_CODE="en")
 class EmailDigestTaskIntegrationTests(TestCase):
 
+    def setUp(self) -> None:
+        self.config = GlobalConfiguration.get_solo()
```

Comment:
please always call the `super().setUp()` method for possible future mixin behaviour.